doc/base/classes.xml

```diff
@@ -12541,14 +12541,22 @@
 </class>
 <class name="ColorRect" inherits="Control" category="Core">
 	<brief_description>
+	Colored rect for canvas.
 	</brief_description>
 	<description>
+	An object that is represented on the canvas as a rect with color. [Color] is used to set or get color info for the rect.
 	</description>
 	<methods>
 		<method name="get_frame_color" qualifiers="const">
 			<return type="Color">
 			</return>
 			<description>
+			Return the color in RGBA format.
```

Comment:
Also missing tab indentation here. Also, the empty newline before the codeblock is not necessary, each line is a paragraph in the XML.